client/client_test.go

```diff
@@ -311,3 +311,13 @@ func (cs *clientSuite) TestDebugGet(c *C) {
 	c.Check(cs.reqs[0].URL.Path, Equals, "/v1/debug")
 	c.Check(cs.reqs[0].URL.Query(), DeepEquals, url.Values{"action": []string{"do-something"}, "foo": []string{"bar"}})
 }
+
+func (cs *clientSuite) TestNonExistentSocketErrors(c *C) {
+	cli, err := client.New(&client.Config{Socket: "/tmp/not-the-droids-you-are-looking-for-bm90IHRoZSBkcm9pZHMgeW91IGFyZSBsb29raW5nIGZvcgo"})
```

Comment:
Love the "not the droids" message. :-) Is the bm90I... stuff needed though?

Reply:
Probably not I was just being overly verbose.